spec/swagger.yaml

```diff
@@ -403,5 +407,5 @@ securityDefinitions:
     tokenUrl: 'https://<servername>.thunderhead.com/one/oauth2token'
     scopes: {}
 # Added by API Auto Mocking Plugin
-host: eu2.thunderhead.com
+host: virtserver.swaggerhub.com
```

Comment:
@andreipop I believe this was auto-updated.  Does this get auto-reverted when it is merged into the master branch?